ext/rbkit_event.c

```diff
@@ -65,6 +66,14 @@ rbkit_object_space_dump_event *new_rbkit_object_space_dump_event(rbkit_object_du
   return event;
 }
 
+rbkit_cpu_sample_event *new_rbkit_cpu_sample_event(rbkit_cpu_sample *sample) {
+  rbkit_cpu_sample_event *event = malloc(sizeof(rbkit_cpu_sample_event));
```

Comment:
We moved away from asserts previously because we don't want to kill the process if rbkit faces exceptions. Maybe we can handle such cases more gracefully, but i'm not sure what's a graceful way to handle these kind of cases.